commercial/app/model/feeds/FeedReader.scala

```diff
@@ -33,7 +33,6 @@ class FeedReader(wsClient: WSClient) extends Logging {
       val requestHolder: WSRequest = {
         val unsignedRequestHolder: WSRequest = wsClient.url(request.url)
           .withQueryString(request.parameters.toSeq: _*)
-          .withRequestTimeout(request.timeout)
```

Comment:
Good point - I'm planning on deleting most of this code soon, but in the meantime I've added in a timeout that is a second longer than the circuit breaker so that, if it doesn't manage to kill the connection then it will timeout after a further second anyway.